packages/e2e-tests/tests/restart-session.test.ts

```diff
@@ -0,0 +1,17 @@
+import { commandPalette, openDevToolsTab, startTest } from "../helpers";
+import { waitForRecordingToFinishIndexing } from "../helpers/utils"
+import test from "../testFixtureCloneRecording";
+
+test.use({ exampleKey: "doc_control_flow.html" });
+
+
+test(`restart debugging session`, async ({
+    pageWithMeta: { page, recordingId },
+    exampleKey,
+}) => {
+    await startTest(page, recordingId);
+    await openDevToolsTab(page);
+    await commandPalette(page, "restart session");
+    await page.waitForNavigation()
+    await waitForRecordingToFinishIndexing(page);
+});
```

Comment:
The test took longer to write than the feature, but i think it's important to try and write tests for new features. Especially infrequently used features like this.

Reply:
Nice!